### DISCUSSION TITLE
kyuubi 多hms 方案，是否支持访问数据湖表，例如paimon、iceberg

### DISCUSSION THREAD

**Discussion by melin** [apache/kyuubi#6636] [Q&A] [answered]
通过spark 访问两个hadoop 集群，数据湖也是用hms catalog，如果使用kyuubi 多hms 方案，是否支持访问数据湖表？
@pan3793

**Comment by pan3793** [comment, marked as answer]
it depends. KSHC currently only supports read/write Hive SerDe tables. under certain criteria, Iceberg tables can be read as Hive SerDe tables, not sure about other lake tables. and I notice Spark made a change recently  to allow V2 catalog support built-in data source, it also changes the situation
